ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -187,7 +187,7 @@ const Post = ({ post }: { post: PostType }) => {
             timestamp={timestamp}
           />
           {headerBlocks.map(headerBlock => (
-            <PostHeadings headerBlock={headerBlock} />
+            <PostHeadings key={headerBlock.id} headerBlock={headerBlock} />
```

Comment:
Are these changes related to the service worker work?

Reply:
No 😅 Just threw another console warning about React needing unique keys. So actually a genuine render bug, but only visible in local development.

Reply:
Hahhaha it's fine, let's leave it in.